src/rapids_pre_commit_hooks/copyright.py

```diff
@@ -305,6 +309,11 @@ def main():
         "the --target-branch argument) to decide whether or not they need a copyright "
         "update."
     )
+    m.argparser.add_argument(
+        "--main-branch",
+        metavar="<main branch>",
+        help="main branch to use instead of branch-<major>.<minor>",
```

Comment:
Let's clarify how `--main-branch` and `--target-branch` relate to one another in the docs. We should clarify that they effectively control the same thing, but with different precedence. Should it be legal to specify both of them?

Should we specify that `<major>` and `<minor>` are assumed to be two digits?

Reply:
I've clarified the docs. It's legal to specify both `--main-branch` and `--target-branch`.

The current implementation does not assume `<major>` and `<minor>` are two digits, they can be any number of numerical digits long.